app/controllers/api/v2/config_reports_controller.rb

```diff
@@ -37,8 +37,9 @@ def show
       param_group :config_report, :as => :create
 
       def create
-        @config_report = ConfigReport.import(params[:config_report], detected_proxy.try(:id))
-        process_response @config_report.errors.empty?
+        import_params = { :report => params[:config_report] }
+        task = ForemanTasks.async_task Actions::Foreman::Report::Import, import_params, ConfigReport, detected_proxy.try(:id)
+        process_success _('Report import has been enqueued, please see the task %s for more details') % task.id
```

Comment:
This comes through POST so we return 201 (:created), would you mind opening redmine issue related to #15779? I'm happy to address.